docs/static/includes/interfaces/int.pe.schema.tf

```diff
@@ -45,7 +57,7 @@ A map of private endpoints to create on the Key Vault. The map key is deliberate
 DESCRIPTION
 }
 
-# Resources
+# The PE resource when we are managing the private_dns_zone_group block:
 resource "azurerm_private_endpoint" "this" {
   for_each                      = var.private_endpoints
```

Comment:
With the existing for_each statement it would need to filter out the private endpoints created where private_endpoints_manage_dns_zone_group is false

Reply:
Thank you. Good catch